wwuatt-ui/src/components/HelloDebora.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div>
+    {{bla}}
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+@Component
+export default class HelloDebora extends Vue {
```

Comment:
O que voce acha de criar um componente nao nomeado?
```suggestion
export default class HelloNameless extends Vue {
```

Reply:
Brincadeira, tava so testando a ferramenta para propor mudança e tal!